### ISSUE TITLE
supply temperature set point tracker

### ISSUE THREAD

**Issue by Mathadon** [ibpsa/project1-boptest#380]
The boptest hydronic HP test case has a PI controller for tracking a zone temperature but not for tracking a supply water temperature. I think it would make sense to add this as an option?

**Comment by dhblum** [comment]
I agree this would makes sense as I think some rule-based control strategies involve supply water temperature reset.

**Comment by JavierArroyoBastida** [comment]
I agree as far as this new addition does not conflict with the current baseline controller. Right now the baseline controller decides on the heat pump modulation signal based on the zone operative temperature reading. Through this process, supply water temperature is indirectly determined. Hence, if a setpoint is defined for supply water temperature, another control logic will have to be defined that determines the supply water temperature setpoint based on the zone operative temperature reading or a heating curve. I don't think there is a possibility for these two controllers to coexist and it needs to be either one or the other, am I wrong? maybe @Mathadon can you send an initial PR just modifying the model to see what you have in mind?
